crates/services/consensus_module/poa/src/sync.rs

```diff
@@ -137,8 +138,7 @@ impl RunnableService for SyncTask {
 
 #[async_trait::async_trait]
 impl RunnableTask for SyncTask {
-    #[tracing::instrument(level = "debug", skip_all, err, ret)]
```

Comment:
Had to remove this from a couple locations since the instrument macro assumes the function returns Result